src/effects/VST/VSTEffect.cpp

```diff
@@ -920,7 +925,8 @@ bool VSTEffect::InitializePlugin()
 {
    if (!mAEffect)
    {
-      Load();
+      if ( ! Load() )
+         Unload();
```

Comment:
This may improve error checking, causing the function to return false of load failes for some other reason after constructing `mAEffect` -- though I see no such reasons now.

I reviewed where else this function is called and I see `VSTEffectsModule::LoadPlugin` does not check the status.  So change that function to check and return null for failure.

Reply:
Now I see that `Load` is not always preceded by `LoadCommon` in the later commits.
But that can't possibly work, can it?  `Load` assumes `mPluginMain` to be inititalized to non-null, but `LoadCommon` is where that happens.